### PR TITLE
Corrected the names referring to WDAC to use App Control

### PR DESCRIPTION
**Pull request by HotCakeX**
# PR Summary

It's no longer called WDAC or Windows Defender Application Control. It's called App Control for Business. Microsoft has updated the relevant docs to use the correct name, i fixed this document to be aligned with them.

https://learn.microsoft.com/en-us/windows/security/application-security/application-control/app-control-for-business/appcontrol

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

**Comment by sdwheeler** [comment]
@vinaypamnani-msft Can you review this PR?

File: reference/docs-conceptual/security/app-control/how-wdac-works.md [hunk 1]
```diff
@@ -1,31 +1,31 @@
 ---
-description: This article explains how WDAC works to secure PowerShell and the restrictions it imposes.
+description: This article explains how App Control works to secure PowerShell and the restrictions it imposes.
```
Comment (vinaypamnani-msft) [inline reference/docs-conceptual/security/app-control/how-wdac-works.md:2]:
```suggestion
description: This article explains how App Control for Business works to secure PowerShell and the restrictions it imposes.
```

File: reference/docs-conceptual/security/app-control/how-wdac-works.md [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
 ms.date: 09/19/2024
-title: How WDAC works with PowerShell
+title: How App Control works with PowerShell
```
Comment (vinaypamnani-msft) [inline reference/docs-conceptual/security/app-control/how-wdac-works.md:4]:
```suggestion
title: How App Control for Business works with PowerShell
```

File: reference/docs-conceptual/security/app-control/how-wdac-works.md [hunk 3]
```diff
@@ -1,31 +1,31 @@
...
-This article explains how **Windows Defender Application Control** (WDAC) secures PowerShell and the
+This article explains how **Application Control for Business** secures PowerShell and the
 restrictions it imposes. The secure behavior of PowerShell varies based on the version of Windows
 and PowerShell you're using.
 
 ## How PowerShell detects a system lockdown policy
 
-PowerShell detects both **AppLocker** and **Windows Defender Application Control** (WDAC) system
-wide polices. AppLocker is deprecated. WDAC is the preferred application control system for Windows.
+PowerShell detects both **AppLocker** and **Application Control for Business** system
```
Comment (vinaypamnani-msft) [inline reference/docs-conceptual/security/app-control/how-wdac-works.md:14]:
```suggestion
PowerShell detects both **AppLocker** and **App Control for Business** system
```

**Review by vinaypamnani-msft — CHANGES_REQUESTED** [review]
Added a couple of suggestions, rest looks good.

**Comment by HotCakeX** [comment]
Thank you

**Pull request merged**